### PR TITLE
perf(license): remove go-enry/go-license-detector.

### PR DESCRIPTION
**Pull request by afdesk**
## Description
There was made a decision to remove `go-enry/go-license-detector`. 
We keep `google/licenseclassifier` only.

it should decrease memory and improve performance.

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: pkg/licensing/classifier.go [hunk 1]
```diff
@@ -67,20 +56,12 @@ func Classify(r io.Reader) ([]types.LicenseFinding, error) {
 	return findings, nil
 }
 
-// FullClassify uses two classifiers to detect and classify the license found in a file
-func FullClassify(filePath string, contents []byte) (types.LicenseFile, error) {
+// GoogleClassify uses Google classifier to detect and classify the license found in a file
```
Comment (knqyf263) [inline pkg/licensing/classifier.go:59]: We can just remove this function and call the existing `Classify` instead.

Comment (knqyf263) [reply pkg/licensing/classifier.go:59]: One second. I might be wrong.

Comment (afdesk) [reply pkg/licensing/classifier.go:59]: ok, i'll try to remove it

Comment (knqyf263) [reply pkg/licensing/classifier.go:59]: I found out those two functions returned different values.

Comment (knqyf263) [reply pkg/licensing/classifier.go:59]: I'll think of that again tomorrow.

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**